frontend/src/tests/pages/PlayPage.test.js

```diff
@@ -343,4 +343,202 @@ describe("PlayPage tests", () => {
             expect(screen.getByTestId("playpage-chat-toggle")).toBeInTheDocument();
         });
     })
-});
\ No newline at end of file
+
+    test("User that has not joined any commons is trying to access an unjoined common", async () => {
+
+        axiosMock.reset();
+        axiosMock.resetHistory();
+        axiosMock.onGet("/api/currentUser").reply(200, { // the only info i need to get is that of the current user.. but do i need to check if the commons exists?
```

Comment:
can you remove this comment?

Reply:
thanks for catching that. just removed it.